### PR TITLE
FW-1743

### PR DESCRIPTION
**Pull request by elysse-fpcc**
This is the detail view for fvbook (words and stories)
It consists of just a presentation layer and uses Guy's data layer on SongStoryData.js

### CODE REVIEW

File: frontend/app/assets/javascripts/components/DetailSongStory/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import DetailSongStoryContainer
+  from 'components/DetailSongStoryStory/DetailSongStoryStoryContainer'
+import DetailSongStoryPresentation
+  from 'components/DetailSongStoryStory/DetailSongStoryPresentation'
+import DetailSongStoryData from 'components/DetailSongStory/DetailSongStoryData'
+
+export default {
+  Container: DetailSongStoryContainer,
```
Comment (gmcauliffe) [inline frontend/app/assets/javascripts/components/DetailSongStory/index.js:8]: NIT - You can remove the import/export of the `DetailSongStoryData`.

File: frontend/app/assets/javascripts/components/DetailSongStory/DetailSongStoryPresentation.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  )
+}
+
+// PROPTYPES
+const {array, func, string, object, number} = PropTypes
+DetailSongStoryPresentation.propTypes = {
+  book: object.isRequired,
+  defaultLanguage: string,
+  intl: object,
+  openBookAction: func,
```
Comment (gmcauliffe) [inline frontend/app/assets/javascripts/components/DetailSongStory/DetailSongStoryPresentation.js:138]: NIT - feel free to remove this prop as it isn't being used.

**Review by gmcauliffe — APPROVED** [review]
Looks great! Take a look at the suggested changes but otherwise good to merge :)

**Comment by elysse-fpcc** [comment]
Thanks for the review @gmcauliffe!

**Pull request merged**